src/components/SalaryInfo/SalaryInfo.jsx

```diff
@@ -0,0 +1,41 @@
+import "./SalaryInfo.scss";
+
+const SalaryInfo = ({ salariesObj }) => {
+  const salariesIntArray = Object.values(salariesObj).map((salary) => {
+    return salary.toLocaleString("en");
+  });
+
+  const titleJSX = Object.keys(salariesObj).map((role) => {
+    return (
+      <h6 key={role} className="salary__title">
+        {role}
+      </h6>
+    );
+  });
+
+  const payJSX = salariesIntArray.map((salary, index) => {
+    return (
+      <h6 key={index} className="salary__pay">
+        £{salary}
+      </h6>
+    );
+  });
+
+  const avgPayJSX = Object.values(salariesObj)
+    .reduce((a, b) => a + b / Object.keys(salariesObj).length)
+    .toLocaleString("en");
+
+  return (
+    <div className="salary">
+      <h5 className="salary__heading">Average Salaries</h5>
```

Comment:
Line 30 h5 tag should not be above a h2 tag, bad for accessibility :(

Reply:
Swapped the h2 and h5 and hard coded the font sizes